src/main/target/STM32_UNIFIED/target.h

```diff
@@ -116,6 +116,8 @@
 
 #define USE_ACCGYRO_BMI270
 
+#define USE_QUADSPI
```

Comment:
This will cause the build to fail for the F4 Unified Targets.

Should be

```
#if defined(STM32F7)
#define USE_QUADSPI
#endif 
```

Reply:
Fixed it.